src/app/payments/razor-pay/razor-pay.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, HostListener, OnInit } from '@angular/core';
+import { PaymentService } from '../../_services/payment.service';
+import { UserService } from '../../_services/user.service';
+import { UserAuthService } from '../../_services/user-auth.service';
+import Swal from 'sweetalert2';
+import { Router } from '@angular/router';
+declare var Razorpay: any;
+
+@Component({
+  selector: 'app-razor-pay',
+  templateUrl: './razor-pay.component.html',
+  styleUrls: ['./razor-pay.component.css']
+})
+export class RazorPayComponent implements OnInit {
+
+  constructor(public userService: UserService, private paymentService: PaymentService, private userAuthService: UserAuthService, private router: Router,) { }
+
+  ngOnInit(): void {
+  }
+
+
+  payNow() {
+    const orderData =  {
```

Comment:
@oveshparasara is this a temporary object? seems the data is static now

Reply:
yes just now static data

Reply:
ok @oveshparasara